.github/workflows/test-conda.yml

```diff
@@ -14,23 +14,60 @@ on:
 
 jobs:
   build-linux:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          # - macOS-latest
+          - ubuntu-latest
+        python-version:
+          - "3.8"
+          - "3.9"
+    env:
+      CI_OS: ${{ matrix.os }}
+      PYVER: ${{ matrix.python-version }}
+      PACKAGE: geometric
 
     steps:
       - uses: actions/checkout@v3
 
-      - name: Add conda to system path
+      - name: Set up conda environment
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: devtools/conda-envs/latest.yaml
+          create-args: >- # beware the >- instead of |, we don't split on newlines but on spaces
+            python=${{ matrix.python-version }}
+
+      # Old version commented out 2023-10-25
+      # - name: Add conda to system path
+      #   run: |
+      #     # $CONDA is an environment variable pointing to the root of the miniconda directory
+      #     echo $CONDA/bin >> $GITHUB_PATH
+      # 
+      # # - name: Configure Conda
+      # #   run: conda config --set channel_priority flexible
+      # - name: Install Mamba
+      #   run: conda install -c conda-forge mamba
+      # 
+      # - name: Install Dependencies
+      #   run: mamba env update --file=devtools/conda-envs/latest.yaml --name=base
+      # 
+      # - name: Activate environment
+      #   run: |
+      #     micromamba activate geometric-test
+
+      - name: Environment Information
+        run: |
+          conda info --all
+          conda list
+
+      - name: Install package
         run: |
-          # $CONDA is an environment variable pointing to the root of the miniconda directory
-          echo $CONDA/bin >> $GITHUB_PATH
-
-      # - name: Configure Conda
-      #   run: conda config --set channel_priority flexible
-      - name: Install Mamba
-        run: conda install -c conda-forge mamba
-        
-      - name: Install Dependencies
-        run: mamba env update --file=devtools/conda-envs/latest.yaml --name=base
+          python -c "import numpy; print(numpy.__version__)"
+          python -m pip install --no-deps .
+          python -c "import numpy; print(numpy.__version__)"
+          python -c "import geometric; print(geometric.__version__)"
```

Comment:
Hopefully already fixed, but for debugging it's often useful to see something like `$ env | grep geometric-test`; if properly activated, the environment name should be in an environment variable (typically `$CONDA_PREFIX`)